plugins/woocommerce-admin/package.json

```diff
@@ -237,22 +224,148 @@
+				"!node_modules/@woocommerce/customer-effort-score/.svn",
+				"!node_modules/@woocommerce/customer-effort-score/src",
+				"!node_modules/@woocommerce/data/node_modules",
+				"!node_modules/@woocommerce/data/.git",
+				"!node_modules/@woocommerce/data/.svn",
+				"!node_modules/@woocommerce/data/src",
+				"!node_modules/@woocommerce/date/node_modules",
+				"!node_modules/@woocommerce/date/.git",
+				"!node_modules/@woocommerce/date/.svn",
+				"!node_modules/@woocommerce/date/src",
+				"!node_modules/@woocommerce/dependency-extraction-webpack-plugin/node_modules",
+				"!node_modules/@woocommerce/dependency-extraction-webpack-plugin/.git",
+				"!node_modules/@woocommerce/dependency-extraction-webpack-plugin/.svn",
+				"!node_modules/@woocommerce/dependency-extraction-webpack-plugin/src",
+				"!node_modules/@woocommerce/experimental/node_modules",
+				"!node_modules/@woocommerce/experimental/.git",
+				"!node_modules/@woocommerce/experimental/.svn",
+				"!node_modules/@woocommerce/experimental/src",
+				"!node_modules/@woocommerce/explat/node_modules",
+				"!node_modules/@woocommerce/explat/.git",
+				"!node_modules/@woocommerce/explat/.svn",
+				"!node_modules/@woocommerce/explat/src",
+				"!node_modules/@woocommerce/internal-js-tests/node_modules",
+				"!node_modules/@woocommerce/internal-js-tests/.git",
+				"!node_modules/@woocommerce/internal-js-tests/.svn",
+				"!node_modules/@woocommerce/internal-js-tests/src",
+				"!node_modules/@woocommerce/internal-style-build/node_modules",
+				"!node_modules/@woocommerce/internal-style-build/.git",
+				"!node_modules/@woocommerce/internal-style-build/.svn",
+				"!node_modules/@woocommerce/internal-style-build/src",
+				"!node_modules/@woocommerce/navigation/node_modules",
+				"!node_modules/@woocommerce/navigation/.git",
+				"!node_modules/@woocommerce/navigation/.svn",
+				"!node_modules/@woocommerce/navigation/src",
+				"!node_modules/@woocommerce/notices/node_modules",
+				"!node_modules/@woocommerce/notices/.git",
+				"!node_modules/@woocommerce/notices/.svn",
+				"!node_modules/@woocommerce/notices/src",
+				"!node_modules/@woocommerce/number/node_modules",
+				"!node_modules/@woocommerce/number/.git",
+				"!node_modules/@woocommerce/number/.svn",
+				"!node_modules/@woocommerce/number/src",
+				"!node_modules/@woocommerce/onboarding/node_modules",
+				"!node_modules/@woocommerce/onboarding/.git",
+				"!node_modules/@woocommerce/onboarding/.svn",
+				"!node_modules/@woocommerce/onboarding/src",
+				"!node_modules/@woocommerce/product-editor/node_modules",
+				"!node_modules/@woocommerce/product-editor/.git",
+				"!node_modules/@woocommerce/product-editor/.svn",
+				"!node_modules/@woocommerce/product-editor/src",
+				"!node_modules/@woocommerce/tracks/node_modules",
+				"!node_modules/@woocommerce/tracks/.git",
+				"!node_modules/@woocommerce/tracks/.svn",
+				"!node_modules/@woocommerce/tracks/src"
```

Comment:
😰

Reply:
As noted above, we can add `files` to these packages for distribution files and it'll clean this up!